docs/en/getting-started/install.md

```diff
@@ -78,8 +78,8 @@ It is recommended to use official pre-compiled `deb` packages for Debian or Ubun
 
 #### Setup the Debian repository
 ``` bash
-sudo apt-get install -y apt-transport-https ca-certificates dirmngr
-sudo gpg --no-default-keyring --keyring /usr/share/keyrings/clickhouse-keyring.gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8919F6BD2B48D754
+sudo apt-get install -y apt-transport-https ca-certificates dirmngr curl gpg
```

Comment:
```suggestion
sudo apt-get install -y apt-transport-https ca-certificates curl gnupg
```

It looks like dirmngr is not needed anymore. And `gpg` is called `gnupg`.

It's much nicer now than it was before, with `GNUPGHOME`

Reply:
You're right. The `dirmngr` doesn't need anymore.